di-ipv-cimit-stub/deploy/template.yaml

```diff
@@ -19,6 +19,14 @@ Parameters:
     Description: The name of the environment to deploy to.
     Type: String
     AllowedPattern: ((production)|(build)|(dev.*))
+  CimitComponentId:
+    Description: "Environment Variables for lambda - Retrieved from SSM Parameter Store"
+    Type: AWS::SSM::Parameter::Value<String>
+    Default: "/stubs/core/cimit/componentId"
+  CimitSigningKey:
```

Comment:
Even though this is a stub, it's bad to potentially expose the signing key in the CloudFormation logs. ECS has the secrets key to help with this but there's no equivalent for Lambdas.
A better approach would be to either
* derive the parameter path names in the code (interpolating using ENVIRONMENT) and then fetch the parameters using Powertools (like core-back does), or
* interpolate the base path (using {ENVIRONMENT}) in the template, pass that as an environment variable, and again retrieve the parameters using Powertools (simpler implementation than core-back)

Reply:
Just checking VC_SIGNING_KEY also defied in ssm for other stubs.